ads.properties.sample

```diff
@@ -0,0 +1,49 @@
+# OAuth refresh token
+#
+# Credential which authorizes access to a Google account. 
+#
+# - See guide for instructions on how to obtain a refresh token.
+#   https://developers.google.com/google-ads/api/docs/oauth/client-library
+# - Can access many Google Ads accounts. 
+# - Operating account is specified by the customerId field in each request. 
+#api.googleads.refreshToken=
+
+# OAuth client ID
+#
+# Specifies the client that is used for Google's OAuth 2 API.
+#
+# - Must use the same client ID which originally generated the refresh token.
+# - Created and managed via http://console.developers.google.com
+# - Doesn't grant access to any specific ad account.
+#api.googleads.clientId=
+
+# OAuth client secret
+#
+# Specifies the client secret that is used for Google's OAuth 2 API. 
+#
+# - Must match the OAuth clientId. 
+# - Created and managed via http://console.developers.google.com
+# - Provides an additional layer of security on top of refresh token.
+#api.googleads.clientSecret=
+
+# Developer token
+# 
+# Specifies the credential used to access the Google Ads API. 
+#
+# - Grants access to the API, not any Google account or Google Ads account.
+# - Associated with a Google Ads manager accoount.
+# - Found by logging into the manager account -> Tools & Settings -> API Center
+# - Must be approved by Google following a review of your intended use. 
+# - Only one approved developer token is permitted per company. 
+# - A test manager account's (unapproved) developer token can be used on test 
+#   accounts. 
+#api.googleads.developerToken=
+
+# Endpoint
+#
+# Specifies the endpoint to use for connecting to the ads API. 
+#
+# - Can be used to connect to a staging endpoint. 
+# - Most external clients can ignore this field.
+#api.googleads.endpoint=
```

Comment:
Have you seen this prove useful for users? If not, I'd prefer we leave it out to avoid confusion.